File: src/integration-test/java/com/fieldju/java/HandlerIntegrationTest.java
package com.fieldju.java;

import org.junit.Before;
import org.junit.Test;

public class HandlerIntegrationTest {

    private Handler handler;

    @Before
    public void before() {
        handler = new Handler();
    }

    @Test
    public void run_handle() {
        handler.handle();
    }
}

File: src/integration-test/scala/com/fieldju/scala/HandlerIntegrationTest.scala
package com.fieldju.scala

import org.junit.{Before, Test}

/**
  * Created by jfiel2 on 2/3/17.
  */
class HandlerIntegrationTest {

  var handler: Handler = _

  @Before
  def before(): Unit ={
    handler = new Handler
  }

  @Test
  def run_handle(): Unit = {
    handler.handle()
  }

}
